openedx/core/lib/api/view_utils.py

```diff
@@ -48,3 +61,60 @@ def handle_exception(self, exc):
             return self.make_validation_error_response(exc)
         else:
             raise
+
+
+def view_course_access(depth=0, access_action='load', check_for_milestones=False):
```

Comment:
The call from `mobile_course_access` passes in `load_mobile` for the access_action.  But I wanted the default to be `load`, as it is used by the majority of callers.